docs-src/typescript/chapter-durable-execution/vscode-debugger-extension.md

```diff
@@ -0,0 +1,17 @@
+---
+id: vscode-debugger-extension
+title: VSCode Debugger extension
+description: Use the Go SDK's static analysis tool to check for non-deterministic code during development.
+sidebar_label: Static analysis
+tags:
+  - typescript sdk
+  - static analysis
+  - vscode debugger extension
+  - determinism
+---
+
+Non-deterministic code can be hard to catch while developing Workflows.
+You can leverage the Temporal TypeScript VSCode extenstion to debug your Workflows by their ID or Workflow Event History file.
+Use the [Temporal Debugger for VS Code](https://marketplace.visualstudio.com/items?itemName=temporal-technologies.temporalio).
+
+See the [vscode-debugger-extension README](https://github.com/temporalio/) for details on how to use it.
```

Comment:
Was the link supposed to be: https://github.com/temporalio/vscode-debugger-extension/blob/main/README.md